src/partials/head-meta.hbs

```diff
@@ -35,9 +43,9 @@
     <meta name="twitter:creator" content="@neo4j" />
     <meta name="twitter:site" content="@neo4j" />
     <meta property="og:type" content="{{{ or page.attributes.postttype 'article' }}}" />
-    {{#with (or page.attributes.ogtitle page.title) }}
-    <meta property="og:title" content="{{{detag this}}} - Neo4j Graph Database Platform" />
-    {{/with}}
+    
+    <meta property="og:title" content="{{#with (or page.attributes.ogtitle page.title) }}{{{detag this}}}{{/with}} - {{#with page.component.title}}{{{detag this}}}{{/with}}" />
```

Comment:
suggestion: We probably want a default title otherwise if `page-ogtitle` is undefined the value will be:

```hbs
<meta property="og:title" content=" - Page Title" />
```

Reply:
Do you mean if no title and no og-title is defined?

Reply:
Oh sorry I missed the `or` condition, never mind!